app/views/api_entreprise/cas_usages/show.html.erb

```diff
@@ -22,7 +22,8 @@
       <div class=role--content"><%= markdown_to_html(@data[:role][:content]) %></div>
     </section>
 
-    <section class="fr-background-default-white">
+    <section class="list_api fr-background-default-white">
+      <h3 id="link_list_api"><%= t('.link_list_api') %></h3>
       <%= render partial: 'api_entreprise/cas_usages/list-api', locals: {current_use_case: params[:uid]} %>
```

Comment:
@Samuelfaure, la section "Liste des api utiles" est actuellement traitée un peu différemment par rapport aux autres sections. Notamment, je ne peux pas ajouter de contenu spécifique pae cas d'usage. Dans mes derniers commits j'ai donc un peu modifié le fonctionnement, en séparant le titre de la section du fichier dédié au tableau des API. Cette manip a fonctionné.

En revanche quand j'ai essayé de créer une clé dédiée pour ajouter mon contenu personnalisé, le navigateur m'a renvoyé une erreur dès que j'ai mis le bloc de code suivant sous le <h3> : 

`<div class=list_api--content"><%= markdown_to_html(@data[:list_api][:content]) %></div>`

Voici le détail de l'erreur : No route matches {:action=>"show", :controller=>"api_entreprise/cas_usages"}, missing required keys: [:uid]
Did you mean?  cas_usage_url

Si jamais c'est facile je veux bien ton éclairage sinon je me débrouillerai sans.

Reply:
Je comprend pas trop ton erreur et j'arrive pas à la reproduire mais on peut se jitsi quand tu veux pour régler ça

Reply:
L'erreur est là https://github.com/etalab/admin_api_entreprise/pull/710/files#diff-7a8dd93caa07327637b05e9c1791cfbbc370ef6b5519990eada8e1a1fa42b496R349

Reply:
@skelz0r je ne comprends pas le lien que tu m'as envoyé, c'est le diff... Du coup je ne sais pas où chercher. Est-ce que tu préciser ? @Samuelfaure j'attends le retour de Loïc, voir si je m'en sors et sinon oui partante pour une visio

Reply:
Iic https://github.com/etalab/admin_api_entreprise/blob/a0b0c5b324d551520bf5d95788cb3538b8d39bc3/config/locales/api_entreprise/cas_usages_entries.fr.yml#L349

Il manque le `uid:` comme sur le reste

Reply:
Merci @skelz0r c'était bien ça même si j'avoue avoir appelé @Samuelfaure à la rescousse car je ne comprenais pas du tout le lien ^^

Ça fonctionne !